docs/user-guide/api-mediation/configuration-logging.md

```diff
@@ -0,0 +1,104 @@
+```
+
+## Default logging configuration file
+
+The following `logback.xml` file is used by default in all API Mediation Layer components:
+
+```xml
+<?xml version="1.0" encoding="UTF-8"?>
+<configuration>
+    <property resource="application.yml" />
+    <property name="MAX_INDEX" value="${rollingPolicy.maxIndex:-12}"/>
+    <property name="MIN_INDEX" value="${rollingPolicy.minIndex:-1}"/>
+    <property name="MAX_FILE_SIZE" value="${rollingPolicy.file.maxSize:-50MB}"/>
+    <property name="STORAGE_LOCATION" value="${apiml.logs.location}" />
+    <property name="apimlLogPattern" value="%d{yyyy-MM-dd HH:mm:ss.SSS,UTC} %clr(&lt;${logbackService:-${logbackServiceName}}:%thread:${PID:- }&gt;){magenta} %X{userid:-} %clr(%-5level) %clr(\\(%logger{15}\\)){cyan} %msg%n"/>
+
+    <turboFilter class="org.zowe.apiml.product.logging.UseridFilter"/>
+    <if condition='property("spring.profiles.include").contains("debug")||property("spring.profiles.include").contains("diag")||property("spring.profiles.include").contains("dev")'>
+        <then>
+        </then>
+        <else>
+            <turboFilter class="org.zowe.apiml.product.logging.ApimlDependencyLogHider"/>
+            <turboFilter class="org.zowe.apiml.product.logging.LogLevelInfoFilter"/>
+            <turboFilter class="ch.qos.logback.classic.turbo.DuplicateMessageFilter">
+                <AllowedRepetitions>0</AllowedRepetitions>
+            </turboFilter>
+        </else>
+    </if>
+    <conversionRule conversionWord="clr" converterClass="org.springframework.boot.logging.logback.ColorConverter" />
+    <appender name="STDOUT" class="ch.qos.logback.core.ConsoleAppender">
+        <encoder class="ch.qos.logback.core.encoder.LayoutWrappingEncoder">
+            <layout class="org.zowe.apiml.product.logging.MaskingLogPatternLayout">
+                <pattern>${apimlLogPattern}</pattern>
+            </layout>
+        </encoder>
+    </appender>
+
+    <appender name="FILE" class="org.zowe.apiml.product.logging.ApimlRollingFileAppender">
+        <file>${STORAGE_LOCATION}/${logbackServiceName}.log</file>
+
+        <rollingPolicy class="ch.qos.logback.core.rolling.FixedWindowRollingPolicy">
+            <fileNamePattern>${STORAGE_LOCATION}/${logbackServiceName}.%i.log</fileNamePattern>
+            <minIndex>${MIN_INDEX}</minIndex>
+            <maxIndex>${MAX_INDEX}</maxIndex>
+        </rollingPolicy>
+
+        <triggeringPolicy class="ch.qos.logback.core.rolling.SizeBasedTriggeringPolicy">
+            <maxFileSize>${MAX_FILE_SIZE}</maxFileSize>
+        </triggeringPolicy>
+
+        <encoder class="ch.qos.logback.core.encoder.LayoutWrappingEncoder">
+            <layout class="org.zowe.apiml.product.logging.MaskingLogPatternLayout">
+                <pattern>${apimlLogPattern}</pattern>
+            </layout>
+        </encoder>
+    </appender>
+
+    <root level="INFO">
+        <appender-ref ref="STDOUT" />
+        <appender-ref ref="FILE" />
+    </root>
+</configuration>
+```
+
```

Comment:
```suggestion

Note: The current configuration you can find for specific module using these links:
- [Gateway](https://github.com/zowe/api-layer/blob/v3.x.x/gateway-service/src/main/resources/logback.xml)
- [ZAAS](https://github.com/zowe/api-layer/blob/v3.x.x/zaas-service/src/main/resources/logback.xml)
- [Caching service](https://github.com/zowe/api-layer/blob/v3.x.x/caching-service/src/main/resources/logback.xml)
- [Other modules](https://github.com/zowe/api-layer/blob/v3.x.x/apiml-common/src/main/resources/logback.xml)

```

It could be helpful. Here are also links for v2:
```
- [Default configuation](https://github.com/zowe/api-layer/blob/v2.x.x/apiml-common/src/main/resources/logback.xml)
- [Could Gateway](https://github.com/zowe/api-layer/blob/v2.x.x/cloud-gateway-service/src/main/resources/logback.xml)
- [Metrics service](https://github.com/zowe/api-layer/blob/v2.x.x/metrics-service/src/main/resources/logback.xml)
- [Caching service](https://github.com/zowe/api-layer/blob/v2.x.x/caching-service/src/main/resources/logback.xml)
```

Reply:
Well, I'm not sure that it's a good idea to provide the links on all logback.xml available, it might confuse customer which to use, espessially, that some are identical or have insignificant difference.